alf/algorithms/actor_critic_algorithm.py

```diff
@@ -45,6 +45,8 @@ def __init__(self,
                  loss_class=ActorCriticLoss,
                  optimizer=None,
                  debug_summaries=False,
+                 summarize_grads_and_vars=False,
+                 summarize_action_distributions=False,
```

Comment:
Right now, summarize_grads_and_vars and summarize_action_distributions are only used by the root algorithm. Perhaps it's better to omit this for all the subclasses to reduce clutter. We can add a function RLAlgorithm.set_summary_setting() to change these flags.

Reply:
How about this?

Reply:
So should we make summarize_grads_and_vars and summarize_action_distribution global flags and maybe set in gin file, or keep the (new commit) PR way to use RLAlgorithm.set_sumary_setting() to set both in Trainer.initialize()?